### PR TITLE
eDiscovery : Update purge beta gesture and add purge to v1 toc

### PR DESCRIPTION
**Pull request by SeunginLyu**
CSDL : 
- [beta](https://msazure.visualstudio.com/One/_git/AD-AggregatorService-Workloads?path=/Workloads/Microsoft.Compliance.Ediscovery/override/schema-Prod-beta.csdl&_a=history)
- [v1](https://msazure.visualstudio.com/One/_git/AD-AggregatorService-Workloads?path=/Workloads/Microsoft.Compliance.Ediscovery/override/schema-Prod-v1.0.csdl)

PR:
- [beta](https://msazure.visualstudio.com/One/_git/AD-AggregatorService-Workloads/pullrequest/7054056)
- [v1](https://msazure.visualstudio.com/One/_git/AD-AggregatorService-Workloads/pullrequest/7115930?_a=files&path=/Workloads/Microsoft.Compliance.Ediscovery/override)

### CODE REVIEW

File: api-reference/v1.0/resources/security-ediscoverypurgedataoperation.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
+---
+title: "ediscoveryPurgeDataOperation resource type"
+description: "Represents the process of purging data of an eDiscovery search."
+author: "SeunginLyu"
+ms.localizationpriority: medium
+ms.prod: "ediscovery"
+doc_type: resourcePageType
+---
+
+# ediscoveryPurgeDataOperation resource type
```
Comment (dodaromike) [inline api-reference/v1.0/resources/security-ediscoverypurgedataoperation.md:10]: Please provide a CSDL PR that introduces this new resource into the v1 endpoint.

File: api-reference/beta/resources/security-ediscoverypurgedataoperation.md [hunk 2]
```diff
@@ -31,6 +31,20 @@ None.
 |resultInfo|[resultInfo](../resources/resultinfo.md)| Contains success and failure-specific result information. |
 |status|[microsoft.graph.security.caseOperationStatus](../resources/security-caseoperation.md#caseoperationstatus-values)| The status of the case operation. Possible values are: `notStarted`, `submissionFailed`, `running`, `succeeded`, `partiallySucceeded`, `failed`.|
 
+### purgeType values
+
+|Name|Description|
+|:----|-----------|
+|recoverable|Purged data is recoverable.|
+|permanentlyDeleted|Purged data is permanently deleted.|
+
```
Comment (dodaromike) [inline api-reference/beta/resources/security-ediscoverypurgedataoperation.md:40]: Please add list item for unknownFutureValue in both enumerations.

File: api-reference/beta/resources/security-ediscoverypurgedataoperation.md [hunk 3]
```diff
@@ -31,6 +31,20 @@ None.
...
+### purgeAreas values
+
+|Name|Description|
+|:----|-----------|
+|mailboxes|Purged data is recoverable.|
+|teamsMessages|Purged data is permanently deleted.|
+
```
Comment (dodaromike) [inline api-reference/beta/resources/security-ediscoverypurgedataoperation.md:47]: Please update these descriptions to reflect the actual members.

File: api-reference/v1.0/resources/security-ediscoverypurgedataoperation.md [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+    "@odata.type": "microsoft.graph.identitySet"
+  },
+  "percentProgress": "Integer",
+  "status": "String",
+  "resultInfo": {
+    "@odata.type": "microsoft.graph.resultInfo"
+  }
+}
+```
+
```
Comment (dodaromike) [inline api-reference/v1.0/resources/security-ediscoverypurgedataoperation.md:77]: Please include changelog entries for all these additions in the PR.

File: api-reference/beta/api/security-ediscoverysearch-purgedata.md [hunk 5]
```diff
@@ -54,7 +54,14 @@ POST /security/cases/ediscoveryCases/{ediscoveryCaseId}/searches/{ediscoverySear
...
 ## Request body
-Do not supply a request body for this method.
+In the request body, supply a JSON representation of the parameters.
+
+The following table shows the parameters that can be used with this action.
+
+|Parameter|Type|Description|
+|:---|:---|:---|
+|PurgeType|String| Options that control whether to the action is soft delete or hard delete. Possible values are `recoverable`, `permanentlydeleted`. |
+|PurgeAreas|String| Options to define the locations to be in scope of the purge action. Possible values are: `mailboxes`, `teamsMessages`. |
```
Comment (isvargasmsft) [inline api-reference/beta/api/security-ediscoverysearch-purgedata.md:64]: Are these new enumerations only used as parameters of this action? If so, please include unknownFutureValue in the list of possible values. If other resources use these enums, please indicate which ones.

**Review by isvargasmsft — CHANGES_REQUESTED** [review]
Hello @SeunginLyu, 

Thank you very much for your work in this PR. 

Please review a few comments Mike and I left, mostly around completeness. 

Please feel free to ping us on the side if you have questions. 

Kind regards, 

Mike and Isaac

File: api-reference/v1.0/api/security-ediscoverysearch-purgedata.md [hunk 6]
```diff
@@ -0,0 +1,90 @@
+---
+title: "ediscoverySearch: purgeData"
+description: "Use the purge data method to delete Teams messages in a eDiscovery search."
+author: "SeunginLyu"
+ms.localizationpriority: medium
+ms.prod: "ediscovery"
+doc_type: "apiPageType"
+---
+
+
```
[the remark this answers is not in the document]
Comment (SeunginLyu) [reply api-reference/v1.0/api/security-ediscoverysearch-purgedata.md:10]: provided in description

File: api-reference/v1.0/resources/security-ediscoverypurgedataoperation.md (hunk 1 above)
Comment (SeunginLyu) [reply api-reference/v1.0/resources/security-ediscoverypurgedataoperation.md:10]: added to PR description

File: api-reference/beta/api/security-ediscoverysearch-purgedata.md (hunk 5 above)
Comment (SeunginLyu) [reply api-reference/beta/api/security-ediscoverysearch-purgedata.md:64]: yes used only for this action

File: api-reference/v1.0/resources/security-ediscoverypurgedataoperation.md (hunk 4 above)
Comment (SeunginLyu) [reply api-reference/v1.0/resources/security-ediscoverypurgedataoperation.md:77]: Trying to get the auto generated changelog

**Comment by SeunginLyu** [comment]
@isvargasmsft Could you take a lookt at the PR again? addressed your feedback.

File: api-reference/beta/api/security-ediscoverysearch-purgedata.md [hunk 7]
```diff
@@ -54,7 +54,15 @@ POST /security/cases/ediscoveryCases/{ediscoveryCaseId}/searches/{ediscoverySear
...
 ## Request body
-Do not supply a request body for this method.
+In the request body, supply a JSON representation of the parameters.
+
+The following table shows the parameters that can be used with this action.
+
+|Parameter|Type|Description|
+|:---|:---|:---|
+|PurgeType|microsoft.graph.security.purgeType| Options that control whether to the action is soft delete or hard delete. Possible values are `recoverable`, `permanentlydeleted`,`unknownFutureValue`. |
+|PurgeAreas|microsoft.graph.security.purgeAreas| Option to define the locations to be in scope of the purge action. Possible values are: `mailboxes`, `teamsMessages`,`unknownFutureValue`. |
```
Comment (isvargasmsft) [inline api-reference/beta/api/security-ediscoverysearch-purgedata.md:64]:
```suggestion
|purgeType|microsoft.graph.security.purgeType| Options that control whether to the action is soft delete or hard delete. Possible values are `recoverable`, `permanentlydeleted`,`unknownFutureValue`. |
|purgeAreas|microsoft.graph.security.purgeAreas| Option to define the locations to be in scope of the purge action. Possible values are: `mailboxes`, `teamsMessages`,`unknownFutureValue`. |
```

**Pull request merged**